src/app/containers/Blocks/index.jsx

```diff
@@ -2,28 +2,22 @@ import React from 'react';
 import { objectOf, arrayOf, func, shape, string, any } from 'prop-types';
 import nanoid from 'nanoid';
 
-// Inlined as this is a temporary component
-const BlockString = props => {
-  const stringProps = JSON.stringify(props);
-  return <p>{stringProps}</p>;
-};
-
 const Blocks = ({ blocks, componentsToRender }) =>
   blocks.map((block, index) => {
     const { type, model } = block;
 
     const { type: typeOfPreviousBlock } = blocks[index - 1] || {};
 
-    const Block = componentsToRender[type] || BlockString;
+    const Block = componentsToRender[type];
 
-    return (
+    return Block ? (
```

Comment:
This is clearer if you return null early & check for a `componentsToRender` and `type` both exist first.

e.g.
```
if (!componentsToRender || !type || !Block) { 
  return null; 
}
```

Also, I know that there is a snapshot with part of it capturing the null case, but it would be much better to have an explicit test case that checks `Blocks` returns exactly `null` when given a random block type.